tests/test_cmdline.py

```diff
@@ -678,29 +707,33 @@ def test_check_json__simple_project__can_report_both_undeclared_and_unused(
         "sources": [
             {
                 "source_type": "CodeSource",
-                "path": f"{tmp_path}/code.py",
+                "path": f"{tmp_path / 'code.py'}",
                 "base_dir": f"{tmp_path}",
             },
             {
                 "source_type": "DepsSource",
-                "path": f"{tmp_path}/requirements.txt",
+                "path": f"{tmp_path / 'requirements.txt'}",
                 "parser_choice": "requirements.txt",
             },
             {
                 "source_type": "PyEnvSource",
-                "path": f"{tmp_path}/my_venv/lib/python{major}.{minor}/site-packages",
+                "path": (
+                    f"{tmp_path / 'my_venv' / 'Lib' / 'site-packages'}"
+                    if platform.system() == "Windows"
+                    else f"{tmp_path}/my_venv/lib/python{major}.{minor}/site-packages"
+                ),
```

Comment:
With the `site_packages()` helper I suggested earlier:

```suggestion
                "path": f"{site_packages(tmp_path / 'my_venv')}",
```

Reply:
Updated in 127473a053bfcf991c9a92f78a3e440f6d84cd0e